extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,31 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!--
+  Guppy enforces the 'white-space', 'height' and 'width' styles causing the non-responsive behaviour.
+  The changes listed below present a plausible solution for the problem, even though it may not
+  agree with the principles of design for this project. !important overrides the properties of 'white-space','height'
+  and 'width' which are given by, thus making the text contained within the box.
+
+  The overriden 'height' property prevents text overflowing out of the textbox vertically. The overriden 'white-space' and 'width'
+  property prevent text overflowing out of the textbox horizontally. Again, specificity overrides are done with !important.
```

Comment:
Some of these are still longer than 80 chars.

Reply:
Indented the lines properly now.